modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -123,6 +129,7 @@ const DEFAULT_PROPS: Props = {
   throttleRequests: true,
   maxRequests: 64,
   maximumMemoryUsage: 32,
+  memoryCacheOverflow: 1,
```

Comment:
Do we need to comment on the value?

Reply:
How about `default values optimized for viewing mesh-based 3D Tiles on both mobile and desktop devices` ?

Reply:
Sounds good.